features/post/post-comment/public-post-comment.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { AnswerProps, PublicPostAnswer } from './answers';
+import { PostComment, PostCommentProps } from './post-comment';
+
+export type PublicPostCommentProps = {
+  answers?: [] | AnswerProps[];
```

Comment:
The `answers` prop for comment replies is used in both public and private comment implementations. Therefore, it makes sense to make the `Answers` component part of the common `PostComment` component. I would put the `Answer` component in the render prop for the public and private versions of the comment. I wrote about the rendering prop [here](https://github.com/Iplatos/inctagram/pull/119#discussion_r1676808574).
Of course, it would be nice to retain the ability to control the behavior of the replies section, for example, to disable its controls etc.